release/release_logs/2.10.0/benchmarks/many_actors.json

```diff
@@ -1,32 +1,32 @@
 {
-    "_dashboard_memory_usage_mb": 514.90816,
+    "_dashboard_memory_usage_mb": 525.451264,
```

Comment:
why 2.10.0 numbers are changed? shouldn't this be like creating a new dir called 2.11.0 or something?